content/en/docs/ambient/usage/troubleshoot-waypoint/index.md

```diff
@@ -38,7 +38,7 @@ Requests to the `reviews` `v2` pod should be enforced by the `reviews-v2-pod-way
     If your source calls the destination using the service's hostname or IP, use the `istioctl experimental ztunnel-config service` command to confirm your waypoint is used by the destination service. Following the example earlier, the `reviews` service should use the `reviews-svc-waypoint` while all other services in the `default` namespace should use the namespace `waypoint`.
```

Comment:
It looks like there are some other instances where we still have experimental for ztunnel-config. Would you be able to take a wider pass on finding these?

```suggestion
    If your source calls the destination using the service's hostname or IP, use the `istioctl ztunnel-config service` command to confirm your waypoint is used by the destination service. Following the example earlier, the `reviews` service should use the `reviews-svc-waypoint` while all other services in the `default` namespace should use the namespace `waypoint`.
```

Reply:
I only focus on https://istio.io/latest/docs/ambient/usage/troubleshoot-waypoint/, it seems all instances are corrected in this page.

Reply:
I would like to see what Ian suggested on a wider pass on finding/resolving these. The PR title suggests this is a case.

Reply:
Just to be clear, I didn't mean the entirety of istio.io needs to be scoured and fixed. I do think at least the page being edited should be corrected entirely though.

Reply:
Yeah, seems like the 4 cases of `experimental` have been updated on this page now @dhawton

Reply:
I scoured all of istio.io and this was all I found